src/Object/EMail.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Friendica\Object;
+
+use Friendica\Object\EMail\IEmail;
+
+/**
+ * The default implementation of the IEmail interface
+ *
+ * Provides the possibility to reuse the email instance with new recipients (@see EMail::withRecipient())
+ */
+class EMail implements IEmail
```

Comment:
It isn't much, granted, but it still is important.
```suggestion
class Email implements IEmail
```